doc/classes/Node.xml

```diff
@@ -623,10 +623,11 @@
 			</return>
 			<argument index="0" name="node" type="Node">
 			</argument>
-			<argument index="1" name="keep_data" type="bool" default="false">
+			<argument index="1" name="keep_groups" type="bool" default="false">
 			</argument>
 			<description>
 				Replaces a node in a scene by the given one. Subscriptions that pass through this node will be lost.
+				When [code]keep_groups[/code] is true, the [code]node[/code] will be added to the same groups that the replaced node was in.
```

Comment:
> If [code]keep_groups[/code] is [code]true[/code], the [code]node[/code] is added to the same groups that the replaced node has been in.

Reply:
squashed that in as well :smile:

Reply:
I was actually suggesting a different phrasing as well as a formatting change. I think it's more correct to put it into a present tense.

Reply:
D'oh :smile:, agreed & fixed.